src/lib/utils.js

```diff
@@ -0,0 +1 @@
+export const testUsers = ['Aeroiio4', 'HhhS', 'Pimdw'];
```

Comment:
seems a bit weird to have a single line utils like this.
I think you can either keep it in the main file for now. Or move the filtering function here instead.

Let's call them `invalidUsers`

Reply:
And add Hshshs, HhaaAh, Fcccf